docs/troubleshooting/troubleshooting.rst

```diff
@@ -0,0 +1,141 @@
+
+Before you commence these steps, please ensure that you have a **tested backup of your Mautic instance** where possible.
+
+Checking for schema updates
+***************************
+
+Mautic has a built-in tool which enables you to verify the database and identify if there are any schema updates required. Visit ``example.com/s/update/schema`` to see if there are any updates required.
+
+If this isn't possible, or your Mautic instance is down completely, follow the next tips.
+
+If you don't have SSH access, skip down to :ref:`SSH access isn't available`.
+
+SSH access is available
+***********************
+
+Having SSH access to your server makes things much easier. Log in via command line, and change directory to the Mautic directory using the command:
+
+.. code:: bash
+
+    cd /your/mautic/directory
+
+1. Try to clear the cache
+=========================
+
+When an upgrade attempt fails in the final step, it may be only the outdated cache that's causing a problem. Use the following command to clear it manually:
+
+.. code:: bash
+
+    php bin/console cache:clear
+
+If this command throws a PHP error, you can try to remove the cache folder using the following command - be careful, this removes all files and folders in the path specified, so ensure you type it correctly, and in the correct directory.
+
+.. code:: bash
+
+    rm -rf var/cache
+
+If clearing the cache hasn't resolved your problems, continue with the next step.
+
+2. Trigger an update manually
+=============================
+
+The first step is to determine if there are any updates available using the following command:
+
+.. code:: bash
+
+    php bin/console mautic:update:find
+
+The output from this command informs you if there are any updates to apply. If there are, run the following command to apply them:
+
+.. code:: bash
+
+    php bin/console mautic:update:apply
+
+If there are no updates found, proceed to the next step.
+
+3. Check for outstanding database migrations
+============================================
+
+Run the following command to identify any outstanding database migrations:
+
+.. code:: bash
+
+    php bin/console doctrine:migration:status
+
+If there are any reported, firstly ensure that you have a tested backup of your database before proceeding, as this command causes changes to the database, then run:
+
+.. code:: bash
+
+    php bin/console doctrine:migration:migrate
+
+4. Try to update the files manually
+===================================
+
+This step requires some manual intervention - there is no command for this part.
+
+To update the files manually, you have to:
+
+   1. Back up (download) all Mautic files from your server to your local computer, using File Transfer Protocol (FTP) or the ``scp`` command, which is much faster.
+   2. Delete all Mautic files and folders on your server. Use FTP or the rm command - use the latter with extreme caution.
+   3. Download the latest Mautic package from :xref:`Mautic Download`.
```

Comment:
This could be replaced with a link to the Composer installation

Reply:
OK let's update this once our Composer docs are more streamlined and user friendly!